core/src/main/java/com/linecorp/armeria/server/DefaultServiceConfigSetters.java

```diff
@@ -239,6 +241,12 @@ public ServiceConfigSetters setHeaders(
         return this;
     }
 
+    public ServiceConfigSetters serviceErrorHandler(ServiceErrorHandler serviceErrorHandler) {
```

Comment:
As this method is exposed in a service builder context, I think, we can drop `service` from the method name for simplicity and consistency with `ServerBuilder.errorHandler()`.
 ```suggestion
    public ServiceConfigSetters errorHandler(ServiceErrorHandler serviceErrorHandler) {
```